features/autorisatie/personen/autorisatie.feature

```diff
@@ -151,12 +152,13 @@ Functionaliteit: autorisatie voor het gebruik van de API
       | burgerservicenummer | 000000024                                    |
       | fields              | geboorte.datum,geboorte.plaats,geboorte.land |
       Dan heeft de response de volgende gegevens
-      | naam     | waarde                                                                  |
-      | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.3             |
-      | title    | U bent niet geautoriseerd voor één of meerdere opgegeven field waarden. |
-      | status   | 403                                                                     |
-      | code     | unauthorizedField                                                       |
-      | instance | /haalcentraal/api/brp/personen                                          |
+      | naam     | waarde                                                                                                    |
+      | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.3                                               |
+      | title    | U bent niet geautoriseerd voor één of meerdere opgegeven field waarden.                                   |
+      | detail   | U bent niet geautoriseerd om de volgende gegevens op te vragen met fields: geboorte.plaats, geboorte.land |
```

Comment:
de fields staan niet in de goede volgorde. De velden worden oplopend gesorteerd. Niet expliciet gespecificeerd, maar conform zoals het voor parameter autorisatie is geïmplementeerd